Library/Formula/xmake.rb

```diff
@@ -0,0 +1,28 @@
+class Xmake < Formula
+  desc "The Automatic Cross-platform Build Tool"
+  homepage "https://github.com/waruqi/xmake"
+  url "https://github.com/waruqi/xmake/archive/v1.0.4.tar.gz"
+  mirror "http://tboox.net/release/xmake/xmake-v1.0.4.tar.gz"
+  sha256 "9d55ca647f6f00bebdb6d9499c6607b71cfa8f42b6539bc0d17a472108f89e60"
+  head "https://github.com/waruqi/xmake.git"
+
+  def install
+    # install to the output directory first
+    system "./install", "output"
+
+    # install the xmake scripts
+    share.mkdir
+    (share/"xmake").mkdir
```

Comment:
You don’t need these two lines, `(share/"xmake").install` will create the directories below.